### ISSUE TITLE
Virtualbox linked base is powered on instead of clones

### ISSUE THREAD

**Issue by AdamKostrzewa** [GNS3/gns3-gui#2889]
I run gns3 on Ubuntu 19.10 (but I also did a test on 18.04 LTS with the same result) for gns3-gui version 2.2.1 (and also with 2.2.0) when working in linked base mode with VirtualBox machines. 
Client creates the clones correctly, but as soon as I want to turn them on. 
the base is powerd on instead of the clones (tested with VirtualBox 6.0.14 and 5.2).
it seems to me that the number in the name of the cloning machine is omitted (-1,-2 etc.).
so I got the following error message for subsequent clones:
"Error when starting XYZ: VirtualBox VM not turned off
VirtualBox VM not switched off".
Thank you in advance for your help!

**Comment by AdamKostrzewa** [comment]
I also got the following error when restarting my GNS3 and trying to open the project with cloned machines

```
=> VirtualBox has returned an error: VBoxManage: error: Could not find a registered machine named 'server-1'

VBoxManage: error: Details: code VBOX_E_OBJECT_NOT_FOUND (0x80bb0001), component VirtualBoxWrap, interface IVirtualBox, callee nsISupports
VBoxManage: error: Context: "FindMachine(Bstr(VMNameOrUuid).raw(), machine.asOutParam())" at line 2620 of file VBoxManageInfo.cpp"
```
server-1 is my cloned machine (server is the name of the linked base)
my log indicates the following:
```

2019-11-02 19:17:45 INFO notification_handler.py:73 New client has connected to controller WebSocket
2019-11-02 19:17:48 INFO project.py:78 Project 09e576cf-7030-4ad5-8618-235ec2346e28 with path '/home/adam/GNS3/projects/test2' created
2019-11-02 19:17:48 INFO web_log.py:233 127.0.0.1 [02/Nov/2019:18:17:48 +0000] "POST /v2/compute/projects HTTP/1.1" 201 299 "-" "Python/3.7 aiohttp/3.6.2"
2019-11-02 19:17:48 INFO __init__.py:112 Executing VBoxManage with command: /usr/bin/VBoxManage --nologo list systemproperties
2019-11-02 19:17:48 INFO virtualbox_vm.py:187 VirtualBox VM 'server-1' [c6a8b4a4-e8cf-4fa7-991a-4fb6587c0872] created
2019-11-02 19:17:48 INFO __init__.py:112 Executing VBoxManage with command: /usr/bin/VBoxManage --nologo showvminfo --machinereadable -- server-1
2019-11-02 19:17:48 ERROR route.py:211 Node error detected: VirtualBoxError
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/gns3_server-2.2.1-py3.7.egg/gns3server/web/route.py", line 196, in control_schema
    await func(request, response)
  File "/usr/local/lib/python3.7/dist-packages/gns3_server-2.2.1-py3.7.egg/gns3server/handlers/api/compute/virtualbox_handler.py", line 63, in create
    adapters=request.json.get("adapters", 0))
  File "/usr/local/lib/python3.7/dist-packages/gns3_server-2.2.1-py3.7.egg/gns3server/compute/base_manager.py", line 259, in create_node
    await node.create()
  File "/usr/local/lib/python3.7/dist-packages/gns3_server-2.2.1-py3.7.egg/gns3server/compute/virtualbox/virtualbox_vm.py", line 189, in create
    vm_info = await self._get_vm_info()
  File "/usr/local/lib/python3.7/dist-packages/gns3_server-2.2.1-py3.7.egg/gns3server/compute/virtualbox/virtualbox_vm.py", line 752, in _get_vm_info
    results = await self.manager.execute("showvminfo", ["--machinereadable", "--", self._vmname])  # "--" is to protect against vm names containing the "-" character
  File "/usr/local/lib/python3.7/dist-packages/gns3_server-2.2.1-py3.7.egg/gns3server/compute/virtualbox/__init__.py", line 125, in execute
    raise VirtualBoxError("VirtualBox has returned an error: {}".format(vboxmanage_error))
gns3server.compute.virtualbox.virtualbox_error.VirtualBoxError: VirtualBox has returned an error: VBoxManage: error: Could not find a registered machine named 'server-1'
VBoxManage: error: Details: code VBOX_E_OBJECT_NOT_FOUND (0x80bb0001), component VirtualBoxWrap, interface IVirtualBox, callee nsISupports
VBoxManage: error: Context: "FindMachine(Bstr(VMNameOrUuid).raw(), machine.asOutParam())" at line 2620 of file VBoxManageInfo.cpp

2019-11-02 19:17:48 INFO web_log.py:233 127.0.0.1 [02/Nov/2019:18:17:48 +0000] "POST /v2/compute/projects/09e576cf-7030-4ad5-8618-235ec2346e28/virtualbox/nodes HTTP/1.1" 409 682 "-" "Python/3.7 aiohttp/3.6.2"
2019-11-02 19:17:48 INFO project.py:361 Project 09e576cf-7030-4ad5-8618-235ec2346e28 with path '/home/adam/GNS3/projects/test2' closed
2019-11-02 19:17:48 INFO web_log.py:233 127.0.0.1 [02/Nov/2019:18:17:48 +0000] "POST /v2/compute/projects/09e576cf-7030-4ad5-8618-235ec2346e28/close HTTP/1.1" 204 223 "-" "Python/3.7 aiohttp/3.6.2"
2019-11-02 19:17:48 INFO web_log.py:233 127.0.0.1 [02/Nov/2019:18:17:48 +0000] "POST /v2/projects/09e576cf-7030-4ad5-8618-235ec2346e28/open HTTP/1.1" 409 626 "-" "GNS3 QT Client v2.2.1"
2019-11-02 19:18:29 INFO web_log.py:233 127.0.0.1 [02/Nov/2019:18:18:29 +0000] "GET /v2/computes HTTP/1.1" 200 1030 "-" "GNS3 QT Client v2.2.1"
```
about my GNS3 
Running GNS3 version 2.2.1 on Linux (64-bit) with Python 3.7.5rc1 Qt 5.13.0 and PyQt 5.13.1.

**Comment by AdamKostrzewa** [comment]
Just checked and GNS3 for windows 10 has the same problem

**Comment by grossmj** [comment]
This should work in soon to be released version 2.2.2. Thanks!
